content/intro-to-storybook/react/zh-TW/conclusion.md

```diff
@@ -1,31 +1,34 @@
 ---
-title: '結論'
-tocTitle: '總結'
-description: '把所有的知識彙總以下，學習更多的 storybook 技巧'
+title: '總結'
+description: '總結已學到的，接著學習更多 Storybook 技巧'
 ---
 
-恭喜! 您在 Storybook 中建立了第一個 UI. 在此過程中,您學習瞭如何構建,組合,測試和部署 UI 元件. 如果您一直關注,您的 repo 和部署的 Storybook 應如下所示:
+恭喜！你已經做好在 Storybook 裡的 UI。一路下來，學到如何打造、組合、測試和發布 UI 元件。如果都照著做，發布的 Storybook 和程式碼應該會長得像這樣：
 
-[📕**GitHub 回購: hichroma / learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
+[📕 **GitHub 專案：chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
-[🌎**部署 Storybook**](https://clever-banach-415c03.netlify.app/)
+[🌎 **已發布的 Storybook**](https://clever-banach-415c03.netlify.app/)
 
-Storybook 是 React,Vue 和 Angular 的強大工具. 它擁有蓬勃發展的開發者社群和豐富的外掛. 這篇介紹揭示了可能的表面. 我們相信,一旦您採用了 Storybook,您將會對構建持久 UI 的效率印象深刻.
+Storybook 在 React、React Native、Vue、Angular、Svelte 和其它 framework 都是很犀利的工具。有熱鬧的開發者社群以及豐富的外掛。這份介紹擷取了其可能性最淺薄的一部份。只要採用 Storybook，我們有自信讓你對效率和打造堅強的 UI 感到驚艷。
 
-## 學到更多
+## 繼續學習
 
-想深入瞭解? 這是有用的資源.
+想要深入探索？這些資源會有幫助。
 
-- [**官方 Storybook 文件**](https://storybook.js.org/docs/react/get-started/introduction)有 API 文件,社群連結和外掛庫.
+- [**Storybook 官方文件**](https://storybook.js.org/docs/react/get-started/introduction)記載 API、社群連結與外掛集散地。
 
-- [**令人愉快的 Storybook 工作流程**](https://www.chromatic.com/blog/the-delightful-storybook-workflow) 重點介紹 Squarespace,Major League Soccer,Discovery Network 和 Apollo GraphQL 的高速團隊使用的工作流程最佳實踐.
+- [**快快樂樂學 Storybook 工作流程**](https://www.chromatic.com/blog/the-delightful-storybook-workflow)隆重介紹 Squarespace、Major League Soccer、Discovery Network 和 Apollo GraphQL 裡高效率團隊的最佳工作流程方法。
 
-- [**視覺測試手冊**](hhttps://storybook.js.org/tutorials/visual-testing-handbook/)深入探討將 Storybook 用於視覺化測試元件. 免費的 31 頁電子書.
+- [**視覺測試手冊**](https://storybook.js.org/tutorials/visual-testing-handbook)深入探討在 Storybook 以視覺測試元件。免費的 31 頁電子書。
 
-## 誰製作了 LearnStorybook.com?
+- 在 [**Discord 聊 Storybook**](https://discord.gg/UUt2PJb) 與 Storybook 社群進行互動。只要你幫助其他 Storybook 使用者，Storybook 使用者也會幫你。
 
-文字,程式碼和製作都是由[Chromatic](https://www.chromatic.com/). 該教程的靈感來自 Chromatic 的流行[GraphQL + React 教程系列](https://www.chromatic.com/blog/graphql-react-tutorial-part-1-6).
+- [**Storybook 部落格**](https://medium.com/storybookjs)展示能夠讓 UI 開發工作流程一致的最新版本與功能。
 
-想要更多這樣的教程和文章嗎? 註冊 Chromatic 郵件列表.
+## LearnStorybook.com 是誰做的？
```

Comment:
@ymcheung if you could adjust the translated text to the equivalent of "Who made the Intro to Storybook tutorial?" we'd appreciate it!